__snapshots__/flex-job.test.js

```diff
@@ -1,6 +1,7 @@
 exports['flex job by specifying a profile and existent jobId should succeed and output default format 1'] = `
 [debug] Checking for package updates
 [debug] Using profile 'profileToGetJobStatus'
+[debug] Error: ENOENT: no such file or directory, open 'projectSetupPath'
```

Comment:
This should be more user-friendly. Something like:
Project configuration file not found: ''filepath'